dags/mapper_dag.py

```diff
@@ -0,0 +1,77 @@
+from datetime import datetime
+import json
+import sys
+
+from airflow.decorators import dag, task
+from airflow.models.param import Param
+
+from rikolti.metadata_mapper.lambda_shepherd import get_vernacular_pages, get_collection
+from rikolti.metadata_mapper.lambda_function import map_page as lambda_map_page
```

Comment:
I think we should try to move away from the use of the word "lambda" in our codebase. Changing the filenames `lambda_shepherd` and `lambda_function` feels like a big shift, but let's not add `lambda_map_page`? The fetcher uses `fetch_page_task` to differentiate between the airflow task and the function `fetch_page` which is doing the work of fetching the page.